tests/go.mod

```diff
@@ -0,0 +1,10 @@
+module compose-spec/tests
+
+go 1.22.0
+
+require (
+	github.com/santhosh-tekuri/jsonschema/v5 v5.3.1
```

Comment:
This is a different library than we use in compose-go, right? Is that one not flexible enough for these tests?

Reply:
Also I wonder if we should have a GHA workflow to run `go test ./tests/...`